templates/test/ci/prow-ci-version/patches/kubeadm-bootstrap.yaml

```diff
@@ -53,8 +53,8 @@
           echo "* downloading package: $$CI_URL/$$CI_CONTAINER.$$CONTAINER_EXT"
           wget "$$CI_URL/$$CI_CONTAINER.$$CONTAINER_EXT" -O "$$CI_DIR/$$CI_CONTAINER.$$CONTAINER_EXT"
           $${SUDO} ctr -n k8s.io images import "$$CI_DIR/$$CI_CONTAINER.$$CONTAINER_EXT" || echo "* ignoring expected 'ctr images import' result"
-          $${SUDO} ctr -n k8s.io images tag k8s.gcr.io/$$CI_CONTAINER-amd64:"$${CI_VERSION//+/_}" k8s.gcr.io/$$CI_CONTAINER:"$${CI_VERSION//+/_}"
-          $${SUDO} ctr -n k8s.io images tag k8s.gcr.io/$$CI_CONTAINER-amd64:"$${CI_VERSION//+/_}" gcr.io/k8s-staging-ci-images/$$CI_CONTAINER:"$${CI_VERSION//+/_}"
+          $${SUDO} ctr -n k8s.io images tag registry.k8s.io/$$CI_CONTAINER-amd64:"$${CI_VERSION//+/_}" registry.k8s.io/$$CI_CONTAINER:"$${CI_VERSION//+/_}"
```

Comment:
This whole section for pulling and tagging images can be dropped, see https://github.com/kubernetes-sigs/cluster-api/pull/6590. Once that merges we may want to see if we can use this capi helper instead of maintaining this ourselves.